.github/workflows/checkstyle.yaml

```diff
@@ -34,11 +34,11 @@ jobs:
     - name: CheckABI
       id: CheckABI
       run: |
-        make checkabi
+        sudo docker run -v $(pwd):/source gmelikov/libabigail make checkabi
     - name: StoreABI
       if: failure() && steps.CheckABI.outcome == 'failure'
       run: |
-        make storeabi
+        sudo docker run -v $(pwd):/source gmelikov/libabigail make storeabi
```

Comment:
Using a container is a neat solution.  However, I think it would be nice if we could instead stick with the packages from the ubuntu repository.  Have you investigated using the ubuntu 20.04 image and install libagigail-1.8.2 from the ubuntu 21.04 repository?  That may not work at all but I wanted to ask.

https://packages.ubuntu.com/hirsute/libabigail-dev

If we do decide to go the container route, let's go ahead and host it at a more official location for OpenZFS.

https://docs.github.com/en/actions/guides/publishing-docker-images

Reply:
We've decided to go with the container.